src/aleph/web/controllers/utils.py

```diff
@@ -19,7 +19,9 @@ def get_pagination_params(request):
         with_pagination = pagination_param != 0
 
         if pagination_page < 1:
-            raise web.HTTPBadRequest(text=f"Query field 'page' must be ≥ 1, not {pagination_page}")
+            raise web.HTTPBadRequest(
+                text=f"Query field 'page' must be ≥ 1, not {pagination_page}"
```

Comment:
Should we add a comment that this counter starts at 1 not 0 ?

Reply:
There's a comment in the new Pydantic model. The end goal here is also to replace this class by a Pydantic model (ex: `PaginatedQuery`) that can be subclassed for any endpoint that requires pagination.